test/Mvc.Api.Analyzers.Test/TestFiles/AddResponseTypeAttributeCodeFixProviderIntegrationTest/CodeFixAddsResponseTypeWhenDifferentFromErrorType.Output.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.Mvc.Api.Analyzers._OUTPUT_
+{
+    [ProducesErrorResponseType(typeof(CodeFixAddsResponseTypeWhenDifferentErrorModel))]
+    [ApiController]
+    [Route("[controller]/[action]")]
+    public class CodeFixAddsResponseTypeWhenDifferentFromErrorType : ControllerBase
+    {
+        [ProducesResponseType(StatusCodes.Status200OK)]
```

Comment:
Btw, I only add the type for 4xx and later status codes.

Reply:
So I assume that means that if you had an action where 200 -> Person, 201 -> Product that wouldn't be inferred/added by the codefix? That seems OK, that's kinda wacky.

Reply:
I could do the 2xx in a follow up. It's a little more involved since we have to look at the method return type and unwrap it (If it's `ActionResult<T>` or `Task<T>`)

Reply:
I think it's fine if we don't, it would be good to capture this limitation somewhere (in a comment).